allennlp/models/model.py

```diff
@@ -333,6 +335,61 @@ def extend_embedder_vocab(self, extended_vocab: Vocabulary) -> None:
             if isinstance(module, (Embedding, TextFieldEmbedder)):
                 module.extend_vocab(extended_vocab)
 
+    def inspection_dict(self,
+                        inspect: str = "parameters",
```

Comment:
I don't like "stringly typed" parameters like this. as I said above, I'm not sure whether you actually want to distinguish the two, but if you do then make two different functions